src/utils/search_pypi.ts

```diff
@@ -0,0 +1,43 @@
+import axios from "axios";
+
+export type PyPIPackageResult = {
+  name: string;
+  version: string;
+  latest_release_number: string;
+  package_manager_url: string;
+};
+
+type PyPiApiProject = {
+  name: string;
+  latest_release_number?: string; // Optional because it might be undefined
+};
+
+export default async function searchPyPIPackages(
+  query: string,
+): Promise<PyPIPackageResult[]> {
+  const apiUrl = `https://libraries.io/api/search`;
```

Comment:
Please do not use a proprietary service I would recommend

caching the results of `curl --header 'Accept: application/vnd.pypi.simple.v1+json' https://pypi.org/simple/ | jq`

or using the bigquery pypi dataset also used here: https://github.com/openteamsinc/Score/blob/main/score/pypi/pypi_downloads.py